corehq/apps/enterprise/views.py

```diff
@@ -73,8 +73,8 @@ def platform_overview(request, domain):
 
     context = get_page_context(
         page_url=reverse('platform_overview', args=(domain,)),
-        page_title=_('Platform Overview for {}').format(request.account.name),
         page_name=_('Platform Overview'),
+        page_title='',
```

Comment:
Looking at `get_page_context`, the `page_name` defaults to whatever is specified in `page_title`, but here we're doing the reverse. The page title also is inserted into the main `<title>` tag. Are we sure this should be empty?

Reply:
Additionally, I was under the impression that it was possible for someone like ops to have multiple enterprise consoles open, so that the "... for X" part of the title was a necessary way to distinguish pages from each other. With these changes, that goes away. Are we sure that doesn't lose functionality?

Reply:
Reverted!